A04_attacks-and-mitigations.md

```diff
@@ -211,8 +211,7 @@ from the authorization server's or the client's website, respectively. Most
 importantly, authorization codes or `state` values can be disclosed in
 this way. Although specified otherwise in [@RFC9110], Section 10.1.3,
 the same may happen to access tokens conveyed in URI fragments due to
-browser implementation issues, as illustrated by Chromium Issue 168213
-[@bug.chromium].
+browser implementation issues, as illustrated by a (now fixed) issue in the Chromium project [@bug.chromium].
```

Comment:
It's still my feeling that this bug is too small to mention. Chrome has had hundreds of security bugs in the past decade, but this document doesn't suggest (for example) that OAuth implementations take additional steps to add randomness to tokens to mitigate [CRIME](https://en.wikipedia.org/wiki/CRIME).